.github/workflows/release-cli.yml

```diff
@@ -77,12 +77,19 @@ jobs:
               --exclude-module=tests --log-level INFO dds_cli/__init__.py
             OUT_FILE_NAME: dds_cli_win_x86_64.exe
             ASSET_MIME: application/vnd.microsoft.portable-executable
-          - os: ubuntu-latest
+          - os: ubuntu-20.04
             TARGET: Linux_x86_64
             CMD_BUILD: >
-              pyinstaller -F -c -n  dds_cli_ubuntu_x86_64 -i resources/scilifelab.icns
+              PyInstaller -F -c -n  dds_cli_ubuntu-20.04_x86_64 -i resources/scilifelab.icns
```

Comment:
Switch to `pyinstaller` here or to `PyInstaller` on the other rows (assuming it doesn't matter and still works)